docs/operations/data_migration.md

```diff
@@ -119,21 +120,22 @@ Later on, we will bundle this file along with the others into its own folder. Ke
 We need to run a few scripts to parse these files into our domain tables.
 We can do this both locally and in a sandbox.
 
-#### STEP 3: Bundle all relevant data files into an archive
+### Step 3: Bundle all relevant data files into an archive
 Move all the files specified in Step 1 into a shared folder, and create a tar.gz.
 
-Create a folder on your desktop called `datafiles` and move all of the obtained files into that. Add these files to a tar.gz archive using any method. See (here)[https://stackoverflow.com/questions/53283240/how-to-create-tar-file-with-7zip].
+Create a folder on your desktop called `datafiles` and move all of the obtained files into that. Add these files to a tar.gz archive using any method. See [here](https://stackoverflow.com/questions/53283240/how-to-create-tar-file-with-7zip).
 
 After this is created, move this archive into `src/migrationdata`.
 
 
-### SECTION 1 - SANDBOX MIGRATION SETUP
+### Set Up Migrations on Sandbox
 Load migration data onto a production or sandbox environment
 
 **WARNING:** All files uploaded in this manner are temporary, i.e. they will be deleted when the app is restaged.
 Do not use these environments to store data you want to keep around permanently. We don't want sensitive data to be accidentally present in our application environments.
 
-#### STEP 1: Using cat to transfer data to sandboxes
+### Step 1: Transfer data to sandboxes
+Use the following cat command to upload your data to a sandbox environment of your choice:
 
 ```bash
 cat {LOCAL_PATH_TO_FILE} | cf ssh {APP_NAME_IN_ENVIRONMENT} -c "cat > /home/vcap/tmp/{DESIRED_NAME_OF_FILE}"
```

Comment:
**Question:** Dumb question, but is there a command for sending not just each file separately, but a whole folder of files at once? Asking as when I was doing this, I was trying to send full folders at once (the tar.gz) and not just each file separately.

ie 
Would `LOCAL_PATH_TO_FILE` be `src/tmp/folder_name_here` and `DESIRED_NAME_OF_FILE` be `folder_name_here`?

Reply:
oops did not see this before merging I'm sorry. I believe generally the file would be a .tar.gz zip file that once unzipped contains multiple files and afterwards the next steps would detail how to unzip and add the migration updates. I believe there is also instructions for if those migrations are via .txt file though I personally never used this method

Reply:
Haha you're okay! I'll ping Zander about it post Thanksgiving.... when I tried the SCP method below I couldn't get the folders or the tar files to send so I've been like ????? if there was something missing 🤣 If I figure it out I'll comment the command I use!